rotkehlchen/assets/asset.py

```diff
@@ -1049,6 +1049,34 @@ def initialize(
         object.__setattr__(asset, 'notes', notes)
         return asset
 
+    @classmethod
+    def deserialize_from_db(
+            cls: Type['CustomAsset'],
+            entry: Tuple[str, str, str, Optional[str]],
+    ) -> 'CustomAsset':
+        """Takes a `custom_asset` entry from DB and turns it into a `CustomAsset` instance."""
+        return cls.initialize(
+            identifier=entry[0],
+            name=entry[1],
+            custom_asset_type=entry[2],
+            notes=entry[3],
+        )
+
+    def serialize_for_db(self) -> Tuple[str, str, Optional[str]]:
+        return (
+            self.identifier,
+            self.custom_asset_type,
+            self.notes,
+        )
+
+    def to_dict(self) -> Dict[str, Any]:
```

Comment:
What do you think about using root's `to_dcit` method to get `identifier` and `asset_type` ? (after the refactor we do this for all assets)

Reply:
The issue is that the `super()` method returns the `asset_type` which is not needed. I could easily `.pop()` but I didn't want to do that.

Reply:
hmmm yeah, perhaps can stay like this then